include/alpaka/rand/TinyMT/Engine.hpp

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2022 Axel Huebl, Benjamin Worpitz, Bernhard Manfred Gruber
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+#pragma once
+
+#include "alpaka/rand/TinyMT/tinymt32.h"
+
+#include <cstdint>
+
+namespace alpaka::rand::engine::cpu
```

Comment:
IMO all files in this folder are not required for Philox and if I see it correctly not tested at all. Please copy only required files from mainline alpaka